src/main/java/org/candlepin/subscriptions/db/model/Offering.java

```diff
@@ -102,19 +102,19 @@
 
   /** Effective physical CPU cores capacity per quantity of subscription to this offering. */
   @Column(name = "physical_cores")
-  private int physicalCores;
+  private Integer physicalCores;
```

Comment:
Fields in Offering are changed from their primitive types to their object types, since the entries are nullable in the DB.